---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* Ph.D in Genetics, Kunming Institute of Zoology, University of Chinese Academy of Sciences, 2020
* B.S. in Biology, Wuhan University, 2014
  * Minor in Computer Science, Wuhan University, 2014

Work experience
======
* June 2024 - present: Postdoctoral Fellowship
  * the University of Hong Kong
  * Duties included: Analysis of sequencing (single-cell/Perturb-seq/bulk RNA-seq/ATAC-seq) and imaging data.
  * Supervisor: Assoc. Prof. Carmen C.L. WONG
  * Achievements:
    * Analyzed single-cell RNA-sequencing data of the tumor mouse model to facilitate the development of combined immune-checkpoint blockade therapy for hepatocyte carcinoma.
    * Analyzed perturb-seq dataset to investigate the epigenetic regulation in cancer cell lines, identified the underlying gene modules, and explored the biological function.

* September 2021 - May 2024: Postdoctoral Fellowship
  * Beijing Genomics Institute (BGI)
  * Duties included: Research (human genetics, population genomics) and scientific support
  * Supervisor: Prof. Xin JIN
  * Achievements:
    * Participated in the drafts of research proposal materials to promote National Genome Projects (for the United Arab Emirates, Israel, Serbia, Saudi Arabia, and Malaysia). Participated in a successful application for an international scientific cooperation grant.
    * Participated in developing a method for screening markers for complex disease using multi-omics data, patent submitted.
    * Participated in resolving the T2T genome reference for the Chinese population, paper published in Cell Research (co-first author).
    * Led and finished the analysis of natural selection of 2q12.3, which affected the immune function of East-Asians, papers under submission (Journal of Genetics and Genomics).

* September 2020 - August 2021: Researcher
  * Zhengbang Group
  * Duties included: Establishing genomics breeding platform
  * Achievements:
    * Established the hardware, as well as a Standard Operating Procedure from sampling to data generation, computing environment, and data analysis pipeline.
    * Generated and processed Chip sequencing data.
        
Skills
======
* Proficient in script language
  * Linux Shell
  * Python
  * R
  * Perl
* Proficient in high-performance computing clusters and clouds.
* Hands-on experience with deep learning tools (PyTorch, scVI, Pyro). Basic knowledge of large language models.
* Proficient in single-cell RNA-seq/multiome dataset analysis and perturb-seq data analysis.
* Proficient in sequencing data (short and long reads) processing.
* Experience in comparative genomics, including:
  * Long-read, Hi-C, and physical-map assisted genome assembly, genome annotation, and phylogenetic analysis.
  * Resolving genetic mechanisms for specific traits with re-sequencing and multi-omics data.
* Proficient in quantitative genetic models and tools.
* English (CET-4 score 565 in 2012, CET-6 score 525 in 2013, GRE score 152+170+3.0 in 2013)
* Chinese (Mandarin, native speaker)

Publications
======
  <ul>{% for post in site.publications reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>




<!--
annotated blocks
Talks
======
  <ul>{% for post in site.talks reversed %}
    {% include archive-single-talk-cv.html  %}
  {% endfor %}</ul>
  
Teaching
======
  <ul>{% for post in site.teaching reversed %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Service and leadership
======
* Currently signed in to 43 different slack teams
-->
